src/client/application.tsx

```diff
@@ -30,5 +30,6 @@ export const render = (element) => {
         document.querySelector(element)
     );
 }
-import { setStatefulModules } from './hmr';
-setStatefulModules('hmr', 'store/', 'actions/');
+import { setStatefulModules } from 'fuse-hmr';
+declare var FuseBox;
+setStatefulModules(FuseBox, ['hmr', 'store/', 'actions/']);
```

Comment:
@nchanged FuseBox loader plugins shared via NPM will need to take `FuseBox` from the user code base. This is the first package sharing FuseBox loader plugins so more patterns will emerge, but worthwhile for you to know :rose: